bigframes/dataframe.py

```diff
@@ -3674,7 +3674,11 @@ def apply(self, func, *, axis=0, args: typing.Tuple = (), **kwargs):
             return result_series
 
         # Per-column apply
-        results = {name: func(col, *args, **kwargs) for name, col in self.items()}
+        if hasattr(func, "bigframes_remote_function"):
```

Comment:
Let's add a test please, we could use the same example you used to report the issue.

Reply:
Added.